module.cake

```diff
@@ -0,0 +1,18 @@
+// Install modules
+#module nuget:?package=Cake.DotNetTool.Module&version=0.3.1
+
+// Install .NET Core Global tools
+#tool "dotnet:https://api.nuget.org/v3/index.json?package=GitVersion.Tool&version=5.0.1"
+
+Task("GitVersion")
+    .Does(context => {
+        context.Information("Calculating Semantic Version");
```

Comment:
Do we really need to specify the `context` here? Can't we just use the extension methods of `ICakeContext`?

```suggestion
        Information("Calculating Semantic Version");
```

Reply:
I personally prefer to use context provided by `Setup/Teardown/Tasks` and not rely on global state.

But I'll change.